components/calendly/new-event.js

```diff
@@ -0,0 +1,41 @@
+const calendly = require("https://github.com/PipedreamHQ/pipedream/components/calendly/calendly.app.js");
+
+module.exports = {
+  name: "New Event",
+  description: "Emits an event for each new event created",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    calendly,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
+    const now = new Date();
```

Comment:
Since you use this code to retrieve the timestamp a month ago in a couple of places, consider abstracting to a method in the app file?